Events.php

```diff
@@ -54,6 +54,12 @@ public static function onRestApiAddRules()
             ['pattern' => 'mail/<messageId:\d+>/entries', 'route' => 'smartVillage/mail/entry/index', 'verb' => 'GET'],
 
             //calendar
+            //Recurring Events endpoints
+            ['pattern' => 'calendar/recurring', 'route' => 'smartVillage/calendar/recurring-events/index', 'verb' => 'GET'],
+            ['pattern'=> 'calendar/container/<containerId:\d+>/recurring', 'route'=> 'smartVillage/calendar/recurring-events/recurring-container','verb'=>'GET'],
+            ['pattern' => 'calendar/entry/<Id:\d+>/recurring','route' => 'smartVillage/calendar/recurring-events/recurring-entry','verb'=>'GET'],
+
```

Comment:
endpoints to create recurring events are missing right?
or is the `RecurrenceFormModel` already existing in the original API which you posted in the issue:

![179174815-6d7b7f2b-f65e-45b0-bee7-5475ae693968](https://user-images.githubusercontent.com/1942953/183872651-a11e7b01-dfe6-4720-bdd9-0d64208bb52c.png)

Reply:
Hi @donni106 
The RecurrenceFormModel is already existing in the original API. Set the correct endDate and try again

Reply:
ok, i think i was able to create recurring events now with the original api.
this was something you found in the documentation somewhere or how did you know, that it is possible like this?

Reply:
i cannot find anything about it in the official docs under https://www.humhub.com/en/marketplace/calendar/docs/swagger/calendar.html#tag/Calendar-entry/paths/~1calendar~1container~1%7Bid%7D/post

Reply:
Hi @donni106 
We will create a controller (CalendarEntryController) and in that controller, we can create the calendar entry with recurring events. is this a better approach?

Reply:
i think it is not necessary as i said that i was able to create recurring events.

Reply:
I suggest you because if you add some condition or functionality later, then it is easier. as you said, if you were able to create recurring events, then it is ok.

Note: we did not alter the original API for creating the event. we just passed some additional data for creating the recurring event.